certbot-apache/certbot_apache/configurator.py

```diff
@@ -822,6 +810,8 @@ def make_vhost_ssl(self, nonssl_vhost):  # pylint: disable=too-many-locals
         # We know the length is one because of the assertion above
         # Create the Vhost object
         ssl_vhost = self._create_vhost(vh_p)
+        if self.conf("handle-sites"):
+            ssl_vhost.enabled = False
```

Comment:
Is this the right logic? I think it works in the conventional case, but what if someone has a file in `sites-enabled` rather than a symlink?